frontend/src/components/managePermissions/table/ApplicationAdminTable.vue

```diff
@@ -110,11 +113,13 @@ const deleteAdmin = (admin: FamAppAdminGetResponse) => {
                 :paginatorTemplate="TABLE_PAGINATOR_TEMPLATE"
                 :currentPageReportTemplate="TABLE_CURRENT_PAGE_REPORT_TEMPLATE"
                 stripedRows
+                :rowStyle="highlightNewUserRow"
             >
                 <template #empty> No user found. </template>
                 <template #loading> Loading users data. Please wait. </template>
                 <Column header="User Name" sortable field="user.user_name">
                     <template #body="{ data }">
+                        <NewUserTag v-if="data.isNewUser" />
```

Comment:
Hi  Jota, I don't think you need to add the isNewUser attribute to the data, you can just check v-if="data.application_admin_id in [your_saved_new_user_access]"

Reply:
Hello! ok, I can do that